lib/matplotlib/pyplot.py

```diff
@@ -1134,7 +1134,7 @@ def figlegend(*args, **kwargs) -> Legend:
 
 @_docstring.dedent_interpd
 def axes(
-    arg: None | tuple[float, float, float, float] = None,
+    arg: None | tuple(float, float, float, float) = None,
```

Comment:
These need to be square brackets, that is type hinting syntax, which does not change because it's a tuple

Reply:
I took the liberty of reverting this via an additional commit in the GitHub UI. Please squash-merge.